src/main/java/com/ait/lienzo/client/core/image/Image.java

```diff
@@ -0,0 +1,366 @@
+package com.ait.lienzo.client.core.image;
+
+import java.util.List;
+
+import com.ait.lienzo.client.core.Attribute;
+import com.ait.lienzo.client.core.Context2D;
+import com.ait.lienzo.client.core.shape.Attributes;
+import com.ait.lienzo.client.core.shape.IDestroyable;
+import com.ait.lienzo.client.core.shape.Shape;
+import com.ait.lienzo.client.core.shape.json.IJSONSerializable;
+import com.ait.lienzo.client.core.shape.json.validators.ValidationContext;
+import com.ait.lienzo.client.core.shape.json.validators.ValidationException;
+import com.ait.lienzo.client.core.types.BoundingBox;
+import com.ait.lienzo.shared.core.types.ShapeType;
+import com.ait.tooling.nativetools.client.collection.MetaData;
+import com.google.gwt.json.client.JSONObject;
+import com.google.gwt.json.client.JSONString;
+
+public class Image
```

Comment:
Class formatted as Kie-Group class, not Lienzo's one. If we want to merge changes to upstream it will be an issue in the future.